reporter/src/main/kotlin/ReportTableModelMapper.kt

```diff
@@ -252,3 +212,43 @@ private fun Project.getScopesForDependencies(
 
     return result
 }
+
+private fun Issue.toResolvableIssue(
+    resolutionProvider: ResolutionProvider,
+    howToFixTextProvider: HowToFixTextProvider
+): ResolvableIssue {
+    val resolutions = resolutionProvider.getIssueResolutionsFor(this)
+    return ResolvableIssue(
+        source = this@toResolvableIssue.source,
+        description = this@toResolvableIssue.toString(),
```

Comment:
The `this@toResolvableIssue.` prefixes can be dropped now as there is no other `this` context than that of `Issue`.